Makefile

```diff
@@ -113,7 +113,7 @@ CDP_TELEMETRY_VERSION ?= ""
 CDP_LOGGING_AGENT_VERSION ?= ""
 
 ifndef JUMPGATE_AGENT_RPM_URL
-	JUMPGATE_AGENT_RPM_URL="https://cloudera-build-us-west-1.vpc.cloudera.com/s3/build/17606589/inverting-proxy/2.x/redhat7/yum/tars/inverting-proxy/jumpgate-agent.rpm"
+	JUMPGATE_AGENT_RPM_URL="https://cloudera-build-us-west-1.vpc.cloudera.com/s3/build/18556518/inverting-proxy/2.x/redhat7/yum/tars/inverting-proxy/jumpgate-agent.rpm"
```

Comment:
you will need to update the builder Jenkins job default parameters for Jumpgate Agent URL as well! cc @pkedvessy  @jszeles

Reply:
Ooh. Thanks for the catch @bergerdenes.
I see that all three of the multi-packer runs spawned by the image-burn task continued to use the older GBN. :(
I would need another run with the correct GBN.

@pkedvessy Can we update or remove this default URL?

Reply:
@zabhi yes I think we can remove the parameter from jenkins so we won't need to update it in both places next time. I'm going to call this out during our daily standup.